Resources/Prototypes/Entities/Clothing/Head/eva-helmets.yml

```diff
@@ -71,22 +71,28 @@
   categories: [ HideSpawnMenu ]
   components:
   - type: Sprite
-    sprite: Clothing/Head/Hardsuits/paramedhelm.rsi
+    sprite: _DV/Clothing/Head/Helmets/paramedhelm.rsi # DeltaV - Custom Vulp Helmet
   - type: HandheldLight
     addPrefix: false
   - type: ToggleableLightVisuals
     spriteLayer: light
+    # Begin DeltaV Additions = Custom Vulp Helmet
     clothingVisuals:
       head:
-      - state: equipped-head-light
+      - state: on-equipped-HELMET
       head-vox:
-      - state: equipped-head-light-vox
+      - state: on-equipped-HELMET-vox
+      head-vulpkanin:
+      - state: on-equipped-HELMET-vulpkanin
   - type: Clothing
     clothingVisuals:
       head:
-      - state: equipped-head
+      - state: off-equipped-HELMET
       head-vox:
-      - state: equipped-head-vox
+      - state: off-equipped-HELMET-vox
+      head-vulpkanin:
+      - state: off-equipped-HELMET-vulpkanin
+    # End DeltaV Additions
```

Comment:
dont do this
just change sprite for the vulp ones specifically and delete the other sprites from the rsi, leave everything else here alone